src/services/axios-wrapper.ts

```diff
@@ -1,11 +1,18 @@
-import axios from 'axios';
+import axios, { AxiosRequestConfig } from 'axios';
 import { store } from '../store'
 import { notLoggedInException } from '../store/user/actions';
 
-const axiosInstance = axios.create();
+const baseUrl = process.env.REACT_APP_MEIZAM_API_BASE_HOST;
+const config: AxiosRequestConfig = {};
```

Comment:
We can set baseURL property to the config of the instance and remove the baseUrl reference in meizam-api.ts